selenium/7.rb

```diff
@@ -0,0 +1,19 @@
+# Write a script to select the i-th dropdown item and return the value
+require 'selenium-webdriver'
+
+# Setting driver path for web browser
+Selenium::WebDriver::Chrome::Service.driver_path = './chromedriver.exe'
+options = Selenium::WebDriver::Chrome::Options.new(args: ['--start-maximized'])
+driver = Selenium::WebDriver.for :chrome, options: options
+
+driver.navigate.to 'https://testpages.herokuapp.com/styled/basic-html-form-test.html'
+
+dropdown_name = 'dropdown'
+dropdown = driver.find_element(:name, dropdown_name)
+
```

Comment:
A better way to interact with dropdowns would be using the Selenium::WebDriver::Support::Select module
`dropdown = Selenium::WebDriver::Support::Select.new(dropdown_element)`

Refer to documentation for more details.